src/System.Data.SqlClient/src/Resources/Strings.resx

```diff
@@ -1059,4 +1062,28 @@
   <data name="AZURESQL_ChinaEndpoint" xml:space="preserve">
     <value>.database.chinacloudapi.cn</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="net_nego_channel_binding_not_supported" xml:space="preserve">
+    <value>No support for channel binding on operating systems other than Windows.</value>
+  </data>
+  <data name="net_gssapi_operation_failed_detailed" xml:space="preserve">
+    <value>GSSAPI operation failed with error - {0} ({1}).</value>
+  </data>
+  <data name="net_gssapi_operation_failed" xml:space="preserve">
+    <value>GSSAPI operation failed with status: {0} (Minor status: {1}).</value>
+  </data>
+  <data name="net_ntlm_not_possible_default_cred" xml:space="preserve">
+    <value>NTLM authentication is not possible with default credentials on this platform.</value>
+  </data>
+  <data name="net_nego_not_supported_empty_target_with_defaultcreds" xml:space="preserve">
+    <value>Target name should be non empty if default credentials are passed.</value>
```

Comment:
Nit: non empty => non-empty

(I realize you probably copied these error messages from the other lib... if you change them, please change them in the original location as well.)